README.md

```diff
@@ -224,3 +224,23 @@ To simulate a production environment, the application should be built with optim
 - Build the library: `npm run build-lib`
 - Build the test app with optimization: `npm run build-app`
 - Access it via <http://dspuiapp.local:8090>
+
+### Publish a new version to NPM
+
+A new version will be published on each github release. It's part of Github actions' workflow. Please follow the steps below to prepare next release:
```

Comment:
I would change this to:
"A new version will be published with each github release it's part of Github actions' workflow. Please follow the steps below to prepare the next release:"

Reply:
Thanks.